spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonDbSchedulerAutoConfiguration.java

```diff
@@ -0,0 +1,97 @@
+import org.axonframework.messaging.ScopeAwareProvider;
+import org.axonframework.serialization.Serializer;
+import org.axonframework.tracing.SpanFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Auto configuration class for the deadline manager and event scheduler using db-scheduler.
+ *
+ * @author Gerard Klijs
+ * @since 4.8.0
+ */
+@AutoConfiguration
+@ConditionalOnBean(Scheduler.class)
+@AutoConfigureAfter(value = {AxonServerAutoConfiguration.class, AxonTracingAutoConfiguration.class},
+        name = {"com.github.kagkarlsson.scheduler.boot.autoconfigure.DbSchedulerAutoConfiguration"})
+public class AxonDbSchedulerAutoConfiguration {
+
+    @Bean
+    public Task<DbSchedulerBinaryEventData> dbSchedulerBinaryEventDataTask() {
+        return DbSchedulerEventScheduler.binaryTask();
+    }
+
+    @Bean
+    public Task<DbSchedulerBinaryDeadlineDetails> dbSchedulerBinaryDeadlineDetailsTask() {
+        return DbSchedulerDeadlineManager.binaryTask();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public EventScheduler eventScheduler(
+            Scheduler scheduler,
+            @Qualifier("eventSerializer") Serializer serializer,
+            TransactionManager transactionManager,
+            EventBus eventBus) {
+        return DbSchedulerEventScheduler.builder()
+                                        .scheduler(scheduler)
+                                        .serializer(serializer)
+                                        .transactionManager(transactionManager)
+                                        .eventBus(eventBus)
+                                        .startScheduler(false)
+                                        .build();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public DeadlineManager deadlineManager(
+            Scheduler scheduler,
+            Configuration configuration,
+            @Qualifier("eventSerializer") Serializer serializer,
+            TransactionManager transactionManager,
+            SpanFactory spanFactory) {
+        ScopeAwareProvider scopeAwareProvider = new ConfigurationScopeAwareProvider(configuration);
+        return DbSchedulerDeadlineManager.builder()
+                                         .scheduler(scheduler)
+                                         .scopeAwareProvider(scopeAwareProvider)
+                                         .serializer(serializer)
+                                         .transactionManager(transactionManager)
+                                         .spanFactory(spanFactory)
+                                         .startScheduler(false)
```

Comment:
Nit: Perhaps add a comment stating why we do not start the scheduler? I can imagine somebody in the future may not understand why this decision is made.